.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: java
 jdk:
+  # JDKs will be maintained by jabba so this entry doesn't do anything
   - oraclejdk8
```

Comment:
This seems like a lot of complexity. Have you instead tried just tweaking this?
```
jdk:
  - oraclejdk8
  - openjdk11
```

Sources: 
 - https://blog.travis-ci.com/support_for_multiple_jdks (Search for `How do I turn it on already?`)
 - https://docs.travis-ci.com/user/reference/trusty/#jvm-clojure-groovy-java-scala-images

Reply:
I tested that at some point as well but IIRC it didn't install the binaries in accessible locations and/or couldn't change the JDKs on the fly properly.